packages/mobile-web/src/tools/mobile-offline/offline-guidance/tool.ts

```diff
@@ -0,0 +1,138 @@
+        outputSchema: this.outputSchema.shape,
+        annotations: annotations,
+      },
+      async () => {
+        try {
+          const reviewInstructions = this.getExpertReviewInstructions();
+
+          return {
+            content: [
+              {
+                type: 'text',
+                text: JSON.stringify(reviewInstructions),
+              },
+            ],
+            structuredContent: reviewInstructions,
+          };
+        } catch (error) {
+          return {
+            isError: true,
+            content: [
+              {
+                type: 'text',
+                text: `Failed to generate review instructions: ${error instanceof Error ? error.message : 'Unknown error'}`,
+              },
+            ],
+          };
+        }
+      }
+    );
+  }
+
+  private getExpertReviewInstructions(): ExpertsReviewInstructionsType {
+    const reviewInstructions: ExpertReviewInstructionsType[] = [
+      this.getConditionalRenderingExpert(),
+      this.getGraphQLWireExpert(),
+    ];
+
+    return {
+      reviewInstructions,
+      orchestrationInstructions:
+        ExpertsReviewInstructionsSchema.shape.orchestrationInstructions.parse(undefined),
+    };
+  }
+
+  private getConditionalRenderingExpert(): ExpertReviewInstructionsType {
+    return {
+      expertReviewerName: 'Conditional Rendering Compatibility Expert',
+      supportedFileTypes: ['HTML'],
+      grounding: dedent`
+        The Komaci offline static analysis engine used by Salesforce Mobile App Plus and Field Service Mobile App 
+        does not support modern conditional rendering directives (lwc:if, lwc:elseif, lwc:else) that were introduced 
+        in newer versions of LWC. These directives must be converted to legacy conditional directives (if:true, if:false) 
+        to ensure compatibility with offline data priming.
+      `,
+      request: dedent`
+        Review the HTML template files for any usage of modern conditional rendering directives:
+        
+        1. Scan for lwc:if, lwc:elseif, and lwc:else attributes on any elements
+        2. For each occurrence, analyze the conditional logic structure
+        3. Determine the appropriate conversion strategy to if:true/if:false syntax
+        4. Consider nested conditional logic and complex boolean expressions
+        5. Validate that the conversion maintains the original functionality
+        6. Report each violation with specific conversion guidance
+        
+        Focus on identifying patterns that use these modern directives and provide actionable 
+        refactoring steps to convert them to legacy directive syntax.
+      `,
+      expectedResponseFormat: {
+        expertReviewerName: 'Conditional Rendering Compatibility Expert',
+        issues: [],
+      },
+    };
+  }
+
+  private getGraphQLWireExpert(): ExpertReviewInstructionsType {
+    return {
+      expertReviewerName: 'GraphQL Wire Configuration Expert',
+      supportedFileTypes: ['JS'],
```

Comment:
In the implementation, should we add the code types to an enum / type?

Reply:
the code type is specified in the zod schema?

```
supportedFileTypes: z
    .array(z.enum(['JS', 'HTML', 'CSS']))
```

or am I misunderstanding what you mean?